conda/cli/main_env_config_vars.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+from argparse import ArgumentParser, Namespace
+
+
+def execute(args: Namespace, parser: ArgumentParser) -> bool:
+    parser.parse_args(["env", "config", "vars", "--help"])
```

Comment:
https://github.com/conda/conda/blob/main/tests/cli/test_conda_argparse.py demonstrates how to test run_command but some of these simpler ones might be testable by a direct call.